packages/bits/demo/src/components/demo/runtime-i18n/runtime-i18n-example.component.ts

```diff
@@ -22,4 +22,8 @@ export class RuntimeI18NExampleComponent {
         localStorage.removeItem("locale");
         this.pageReload();
     }
+
+    public ngOnDestroy() {
+        localStorage.removeItem("locale");
```

Comment:
My browser permanently had this locale item in local storage because I had visited this page and played with the controls once. This is to fix that problem.